src/MPQEditor/MPQEditor.ts

```diff
@@ -269,6 +271,75 @@ export class MPQEditorProvider implements vscode.CustomTextEditorProvider {
     //TODO process messages
   }
 
+  /**
+   * @brief Initializes webviewpanel
+   */
```

Comment:
IMHO, this comment is needless. please remove.

Reply:
Ok.
@seanshpark @dayo09 
done.